doc/nrf/config_and_build/multi_image.rst

```diff
@@ -230,25 +243,14 @@ For example, add ``--trace-expand`` to that variable to output more debug inform
 
 With west, you can pass these configuration variables into CMake by using the ``--`` separator:
 
-.. code-block:: console
+  .. code-block:: console
 
-   west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- \
-   -Dmcuboot_CONF_FILE=prj_a.conf \
-   -DCONF_FILE=app_prj.conf
-
-You can make a project pass Kconfig configuration files, fragments, and devicetree overlays to child images by placing them in the :file:`child_image` folder in the application source directory.
-The listing below describes how to leverage this functionality, where ``ACI_NAME`` is the name of the child image to which the configuration will be applied.
-
-.. literalinclude:: ../../../cmake/multi_image.cmake
-    :language: c
-    :start-at: It is possible for a sample to use a custom set of Kconfig fragments for a
-    :end-before: set(ACI_CONF_DIR ${APPLICATION_CONFIG_DIR}/child_image)
+     west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- \
+     -Dmcuboot_CONF_FILE=prj_a.conf \
+     -DCONF_FILE=app_prj.conf
 
-Variables in child images
--------------------------
-
-It is possible to provide configuration settings for child images, either as individual settings or using Kconfig fragments.
-Each child image is referenced using its image name.
+Examples of commands
```

Comment:
Maybe we can call it `Child image Kconfig modification` (for consistency with the subsequent section about the DTS overlays)